### ISSUE TITLE
JS broken when LOCALISE_TIMESTAMP set to False

### ISSUE THREAD

**Issue by volans-** [voxpupuli/puppetboard#461]
In [_macros.html#L71](https://github.com/voxpupuli/puppetboard/blob/master/puppetboard/templates/_macros.html#L71) there is a call to `localise_timestamp()` that is done independently of the `LOCALISE_TIMESTAMP` setting. When `LOCALISE_TIMESTAMP` is set to `False`, `moment.js` is not loaded at all (see `layout.html`) resulting in JS errors:
```
Uncaught TypeError: $(...).localise_timestamp is not a function
```
Ideally even when `LOCALISE_TIMESTAMP` is set to `False` the timestamps should be properly formatted in a nice way, just preserving the original times without TZ conversion.

As a side note the `README` is a bit confusing regarding this option given that it states:

> LOCALISE_TIMESTAMP: Normalize time based on localserver time.

While it seems to me that it normalize it to the client's time when set to `True`.

**Comment by emlun007** [comment]
moment.js should be loaded in any case to get proper formatting of the data:

```
{% if config.OFFLINE_MODE %}

      <script src="{{ url_for('static', filename='jquery-2.1.1/jquery.min.js') }}"></script>
      <script src="{{ url_for('static', filename='jquery-datatables-1.10.13/jquery.dataTables.min.js') }}">
      <script src="{{ url_for('static', filename='jquery-datatables-1.10.13/dataTables.semanticui.min.js') }}">
      <script src="{{ url_for('static', filename='moment.js-2.7.0/moment.min.js') }}"></script>

```

So to get a nice formatting, just without TZ conversion, in timestamps.js you can just remove the line:
`      result.local();`

to get the data in utc, so something like: 

```
$.fn.extend({
  localise_timestamp: function (){
    var tstring = $(this).text().trim();
    if (tstring === "None"){
      $(this).text('Unknown');
    } else {
      var result = moment(tstring).utc();
      $(this).text(result.format('MMM DD YYYY - HH:mm:ss'));
    }
  }
})
```

**Comment by gdubicki** [comment]
Hi @volans-!

The code you mentioned has been changed in https://github.com/voxpupuli/puppetboard/pull/708 but I noticed today that `LOCALISE_TIMESTAMP=False` still did not work.

But I created https://github.com/voxpupuli/puppetboard/pull/722 that should make it work.

**Comment by volans-** [comment]
@gdubicki Thanks for the follow up.

**Comment by gdubicki** [comment]
Fix released in [v4.2.1](https://github.com/voxpupuli/puppetboard/releases/tag/v4.2.1) today.
